### PR TITLE
RD-255: Add claimed members for TPP licenses implementation.

### PR DESCRIPTION
**Pull request by VKCR**

### CODE REVIEW

File: src/http/HttpClient.js [hunk 1]
```diff
@@ -51,14 +51,19 @@ class HttpClient {
      * Creates a memberId.
      *
      * @param  {string} memberType - type of member to create. 'PERSONAL' if undefined
+     * @param  {string} tokenRequestID - token request id if the member is being claimed
```
Comment (sam-token) [inline src/http/HttpClient.js]: nit: `tokenRequestId`
Indicate optional here too.

File: src/main/TokenIO.js [hunk 2]
```diff
@@ -212,6 +214,23 @@ export class TokenIO {
         return this.createMember(alias, CryptoEngine, 'BUSINESS');
     }
 
+    /**
+     * Creates a claimed member with a alias and a keypair, using the provided engine
+     *
+     * @param  {Object} alias - alias to set for member,
+     *                  falsy value or empty object for a temporary member without an alias
```
Comment (sam-token) [inline src/main/TokenIO.js]: will an alias ever be necessary in createClaimedMember?

File: src/http/HttpClient.js (hunk 1 above)
Comment (VKCR) [reply src/http/HttpClient.js]: done

File: src/main/TokenIO.js (hunk 2 above)
Comment (VKCR) [reply src/main/TokenIO.js]: In the current appless case, no, but maybe in the future?

**Pull request merged**